sorts/short_quicksort.py

```diff
@@ -0,0 +1,21 @@
+def QuickSort(A):

```

Comment:
What is `A`? Why CamelCase in python? Please add doctests, typehints,description then we will review logic.

Reply:
I am sorry, i think for to change the name of the formal parameter. I am not much experienced as of now, in a learning stage, didnt know about conventional python casing. Will review it and re-request.
Thanks